docs/docs/dev/policies/traffic-log.md

```diff
@@ -164,7 +164,7 @@ When `backend ` field of a `TrafficLog` policy is omitted, the logs will be forw
 
 **1. Install Loki**
 
-To install Loki use `kumactl install logging | kubectl apply -f -`. This will deploy Loki automatically in a `kuma-logging` namespace.
+To install Loki use `kumactl install observability --components loki | kubectl apply -f -`. This will deploy Loki automatically in a `mesh-observability` namespace.
```

Comment:
I think the whole point of this effort was to simplify all this with a common install and not have fun doing selective stuff. So we'd always say something like: "install the observability" and link to the doc for this.